### ISSUE TITLE
2.0.20123: GameParser#newGameParseException:118 - GameParseException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#6748]
## User Description
I just installed the new TripleA version and got this error:

## Log Message
Could not parse:jar:file:/C:/Users/Rick%20B/triplea/downloadedMaps/napoleonic_empires-master.zip!/napoleonic_empires-master/map/games/Napoleonic_Empires_FFA_5player.xml

## TripleA Version
2.0.20123

## Java Version
11.0.6

## Operating System
Windows 10

## Exception 
games.strategy.engine.data.GameParseException: map name: 'jar:file:/C:/Users/Rick%20B/triplea/downloadedMaps/napoleonic_empires-master.zip!/napoleonic_empires-master/map/games/Napoleonic_Empires_FFA_5player.xml', game name: 'Napoleonic Empires: FFA 5 Player', Missing property definition for option 'costPU' in attachment 'politicalActionAttachment_Treaty_France_Ceasefire_UnitedKingdom'

## Stack Trace
```
games.strategy.engine.data.GameParseException: map name: 'jar:file:/C:/Users/Rick%20B/triplea/downloadedMaps/napoleonic_empires-master.zip!/napoleonic_empires-master/map/games/Napoleonic_Empires_FFA_5player.xml', game name: 'Napoleonic Empires: FFA 5 Player', Missing property definition for option 'costPU' in attachment 'politicalActionAttachment_Treaty_France_Ceasefire_UnitedKingdom'
	at games.strategy.engine.data.GameParser.newGameParseException(GameParser.java:118)
	at games.strategy.engine.data.GameParser.newGameParseException(GameParser.java:111)
	at games.strategy.engine.data.GameParser.lambda$setOptions$12(GameParser.java:1378)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at games.strategy.engine.data.GameParser.setOptions(GameParser.java:1376)
	at games.strategy.engine.data.GameParser.parseAttachment(GameParser.java:1322)
	at games.strategy.engine.data.GameParser.parseAttachments(GameParser.java:1236)
	at games.strategy.engine.data.GameParser.parseMapDetails(GameParser.java:212)
	at games.strategy.engine.data.GameParser.parse(GameParser.java:106)
	at games.strategy.engine.data.GameParser.parse(GameParser.java:98)
	at games.strategy.engine.data.GameParser.parse(GameParser.java:84)
	at games.strategy.engine.framework.ui.DefaultGameChooserEntry.getCompleteGameData(DefaultGameChooserEntry.java:63)
	at games.strategy.engine.framework.ui.DefaultGameChooserEntry.fullyParseGameData(DefaultGameChooserEntry.java:47)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.loadDefaultGameSameThread(GameSelectorModel.java:194)
	at games.strategy.engine.framework.GameRunner.loadGameIfSpecified(GameRunner.java:98)
	at games.strategy.engine.framework.GameRunner.showMainFrame(GameRunner.java:83)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

**Comment by DanVanAtta** [comment]
If you're able to update the map, or delete the map file and then redownload it, the problem should be resolved.

We certainly have a migration dilemna here as the 1.9 only attribute is going to be there for newly migrating players.

**Comment by DanVanAtta** [comment]
This should be fixed in 2.0.20183
